marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -482,8 +482,23 @@ function handleDuplicateSources (propToValidate, arrayWithDuplicates) {
   return deDupedArray;
 }
 
+function addDocumentMetadataToSearchResults(searchResponse) {
+  searchResponse.results.forEach(result => {
+    let metadata = {};
+    let documentMetadata = xdmp.documentGetMetadata(result.uri);
+    if(documentMetadata) {
+      metadata["Flow"] = documentMetadata.datahubCreatedInFlow ? documentMetadata.datahubCreatedInFlow : "";
```

Comment:
While we don't want to change the labels yet in the UI, I think we should start using accurate names in the code. We can't change the names in the data, but for in-memory data and variables, we can use correct names. So:

- lastProcessedByFlow
- lastProcessedByStep
- lastProcessedDateTime

That should go into the Java class that the controller uses to generate Swagger too. The UI team will see them and at least understand what the values represent.